realm/realm-library/src/main/cpp/io_realm_internal_TableQuery.cpp

```diff
@@ -19,9 +19,11 @@
 #include <shared_realm.hpp>
 #include <object_store.hpp>
 #include "util.hpp"
+#include "jni_util/java_local_ref.hpp"
 #include "io_realm_internal_TableQuery.h"
 
 using namespace realm;
+using namespace realm::jni_util;
```

Comment:
I see I misread JniUtils:: for jni_utils:: - so there was no real inconsistency. 
It was just a detail - do as you please. It will work either way :-)